pages/linux/tlp.md

```diff
@@ -1,6 +1,6 @@
 # tlp
 
-> Advanced power management for Linux. See `tlp-stat` page for additional information.
+> Power management for Linux. See `tlp-stat` page for additional information.
```

Comment:
I'd perhaps suggest that in this case we should keep the `advanced`. It differentiates the tool from existing basic power management features built-in to one's operating system. It suggests that additional more advanced features are provided by the command, which is the whole aim of the command itself.

Reply:
While writing this I've also thought of that. I'm changing it because I feel neutral about this.